src/main/java/frc/robot/Robot.java

```diff
@@ -4,33 +4,22 @@
 
 package frc.robot;
 
-import java.util.List;
-
-import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.wpilibj.TimedRobot;
-import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
 import frc.robot.commands.AutoCommandInterface;
 import frc.robot.commands.NoteAuto;
-import frc.robot.subsystems.DriveTrain;
 
 public class Robot extends TimedRobot {
     private AutoCommandInterface m_autonomousCommand;
     private AutoCommandInterface m_prevAutoCommand = null;
     
-    private SendableChooser<AutoCommandInterface> m_chosenAuto = new SendableChooser<>();
-
     private RobotContainer m_robotContainer;
 
     @Override
     public void robotInit() {
         m_robotContainer = new RobotContainer();
-        DriveTrain driveTrain = m_robotContainer.getDriveTrain();
-        // Initialize the list of available Autonomous routines
-        // m_chosenAuto.setDefaultOption("Test Auto", new NoteAuto(driveTrain));
 
-        SmartDashboard.putData("Chosen Auto", m_chosenAuto);
+        m_robotContainer.initAutoSendableChooser();
```

Comment:
Why not just call this in the RobotContainer constructor, just like configureBindings() ?